cloudify_gcp/compute/instance.py

```diff
@@ -249,26 +274,27 @@ def list(self):
 
         return self.discovery.instances().list(
             project=self.project,
-            zone=self.zone).execute()
+            zone=basename(self.zone)).execute()
 
     def to_dict(self):
         def add_key_value_to_metadata(key, value, body):
             body['metadata']['items'].append({'key': key, 'value': value})
 
-        def get_instance_ssh_keys():
-            agent_key = utils.get_agent_ssh_key_string()
```

Comment:
Will fix this in `utils.get_agent_ssh_key_string()` so shouldn't need changing here